test/image/mocks/sunburst_pie_cartesian.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Nicely done. 
Could we reuse this mock instead?
test/image/baselines/display-text_zero-number.png

Reply:
What's wrong with the mock I used?

Reply:
Ha I see, that mock also has a sunburst along with cartesian traces. Yep, good call. Thanks!

Reply:
Done in https://github.com/plotly/plotly.js/pull/3860/commits/d5064ff3e7da3b036b66e4def205e558d36367cf